protocol/chainlib/common_test_utils.go

```diff
@@ -179,7 +178,7 @@ func SetupForTests(t *testing.T, numOfProviders int, specID string, getToTopMost
 	msg, err := stakingtypes.NewMsgCreateValidator(
 		sdk.ValAddress(ts.Validator.Addr),
 		ts.Validator.PubKey,
-		sdk.NewCoin(epochstoragetypes.TokenDenom, sdk.NewIntFromUint64(uint64(balance))),
+		sdk.NewCoin(ts.Keepers.StakingKeeper.BondDenom(sdk.UnwrapSDKContext(ts.Ctx)), sdk.NewIntFromUint64(uint64(balance))),
```

Comment:
can you create a function for ease of use?
ts.BondDenom?

Reply:
Is this what you meant? ffed9418290964792abd2a24222d633678b89ded